### DISCUSSION TITLE
我是如何减少我的鼠标使用的？ - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#25] [Announcements]
# 我是如何减少我的鼠标使用的？ - forecho's Blog

引言
虽然鼠标是一个伟大的发明，但是作为一种非常方便的输入设备，它也有一些缺点，比如说：

没有键盘效率高
会让你得「鼠标手」

所以我觉得还是日常使用电脑工作较多的职业还是多用键盘比较好。

https://blog.forecho.com/how-i-reduced-my-mouse-use.html

**Comment by songshua295** [comment]
正好在看hugo的博客配置，在网站中看到的你的博客，没想到居然这个时代还有更新博客如此勤快的，我来评论一下，冒个泡。鼠标手一般打游戏比较容易，或者是做设计的时候容易，我之前狠敲代码的时候，频繁操作手也会有一些感觉。后来我发现**通过将手肘靠着桌面边缘，将手前端抬高一点**对防止鼠标手有不错的效果。频繁操作鼠标的时候可以试试看

**Comment by sunxyz** [comment]
推荐一个 rss阅读器 支持 键盘事件无障碍（快捷键）

left right up down 在订阅源和文章列表文章之间跳转
a d 文章切换
w s 文章滚动
q 退出 e 内嵌视图 r 阅读视图 t 打开源网站

[webfollow.cc](https://webfollow.cc)
